trino/client.py

```diff
@@ -747,6 +746,14 @@ def __init__(
         self._legacy_primitive_types = legacy_primitive_types
         self._row_mapper: Optional[RowMapper] = None
 
+    @property
+    def query_id(self) -> Optional[str]:
+        return self._query_id
+
+    @property
+    def sql(self) -> Optional[str]:
```

Comment:
Perhaps a better name is `query`, it will follow the convention as used in `query_id`. `Psycopg` follow this as well.